platforms/corda-ent/charts/idman/values.yaml

```diff
@@ -71,84 +61,117 @@ vault:
+  # Java class name to use for the database
+  # Eg. driverClassName: "org.h2.Driver"
   driverClassName:
-  # Ex. url: "jdbc:h2:file:./h2/identity-manager-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
+  # The DB connection URL 
+  # Eg. url: "jdbc:h2:file:./h2/identity-manager-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
   url: 
-  # Ex. user: "example-db-user"
-  user: 
-  # Ex. password: "example-db-password"
+  # DB user
+  # Eg. user: "example-db-user"
+  user:
+  # DB password 
+  # Eg. password: "example-db-password"
   password: 
-  # Ex. runMigration: "true"
+  # Migrations of database can be run as part of the startup of Idman, if set to true. 
+  # If set to false, it will be run prior to setting up the Idman.
+  # Eg. runMigration: "true"
   runMigration: 
 
-# Specify the maximum size of the memory allocation pool
-# cordaJarMx: 1
-cordaJarMx: 
-
-# healthCheckNodePort set to get rid of logs pollution
-# Ex. healthCheckNodePort: 0
-healthCheckNodePort: 
+#############################################################
+#                       Settings                            #
+#############################################################
+# Provide volume related specifications
+volume:
+  # Eg. baseDir: /opt/corda
+  baseDir: 
 
-# where is CENM service jar file stored
-# Ex. bin
+# Provide the path where the CENM Idman .jar-file is stored
+# Eg. jarPath: bin
 jarPath: 
 
-# where is CENM service configuration file stored
-# Ex. etc
+# Provide the path where the CENM Service configuration files are stored
+# Eg. configPath: etc
 configPath: 
 
-# Sleep time after any error encountered in starting idman
-# Ex. 120
+# Provide any extra annotations for the PVCs
+pvc:
+  #   annotations:
+  #     key: "value"
+  annotations: {}
+
+# Provide any extra annotations for the deployment
+deployment:
+  #   annotations:
+  #     key: "value"
+  annotations: {}
+
+# Specify the maximum size of the memory allocation pool
+# Eg. cordaJarMx: 1
+cordaJarMx: 
+
+# Provide the number of replicas for your pods
+# Eg. replicas: 1
+replicas:
+
+# Sleep time in seconds, occurs after any error is encountered in start-up
+# Eg. 120
 sleepTimeAfterError:
 
-healthcheck:
-  #Provide the interval in seconds you want to iterate till db to be ready
-  #Eg. readinesscheckinterval: 10
-  readinesscheckinterval: 
-  #Provide the threshold till you want to check if specified db up and running
-  #Eg. readinessthreshold: 15
-  readinessthreshold:
+# Values    
+healthCheck:
+  # Provide the interval in seconds you want to iterate till db to be ready
+  # Eg. readinessCheckinterval: 10
+  readinessCheckInterval: 
+  # Provide the threshold till you want to check if specified db up and running
+  # Eg. readinessThreshold: 15
+  readinessThreshold:
+  # Health Check node port set to get rid of logs pollution
+  # Eg. nodePort: 0
+  nodePort: 
+  
```

Comment:
newline to be fixed

Reply:
Removed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5)